src/fava/json_api.py

```diff
@@ -288,6 +288,13 @@ def put_source_slice(entry_hash: str, source: str, sha256sum: str) -> str:
     return g.ledger.file.save_entry_slice(entry_hash, source, sha256sum)
 
 
+@api_endpoint
+def delete_source_slice(entry_hash: str, sha256sum: str) -> str:
+    """Delete an entry source slice."""
+    g.ledger.file.delete_entry_slice(entry_hash, sha256sum)
+    return f"Delete entry {entry_hash}."
```

Comment:
```suggestion
    return f"Deleted entry {entry_hash}."
```

Reply:
I corrected that and another typo that I missed before